### DISCUSSION TITLE
[@types/styletron-standard] animationName has wrong typings. not accepting object

### DISCUSSION THREAD

**Discussion by hhvys** [DefinitelyTyped/DefinitelyTyped#60783] [Issues with a @types package]
https://styletron.org/concepts#keyframes

according to docs animationName can be an object. But typescript is not recognizing this. We should update types for animationName similar to what we have in flow types [here](https://github.com/styletron/styletron/blob/master/packages/styletron-standard/src/style-types.js)

[a comment by an automated account is not shown]

**Comment by erictaylor** [comment]
Feel free to submit a PR and I’ll happily take a look at it.
